src/components/keymap/keymap.component.ts

```diff
@@ -1,47 +1,32 @@
-import { Component, ViewChildren, QueryList, ElementRef, OnInit, AfterViewInit, Renderer } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
 import { ActivatedRoute } from '@angular/router';
 
-import { SvgKeyboardPopoverComponent } from '../svg/popover';
+import { SvgKeyboardWrapComponent } from '../svg/wrap';
 import { Layers } from '../../../config-serializer/config-items/Layers';
 import { UhkConfigurationService } from '../../services/uhk-configuration.service';
 import { Keymap } from '../../../config-serializer/config-items/Keymap';
 import { Subscription } from 'rxjs/Subscription';
+import { LayersComponent } from '../layers';
 
 @Component({
     selector: 'keymap',
     template: require('./keymap.component.html'),
     styles: [require('./keymap.component.scss')],
-    directives: [SvgKeyboardPopoverComponent],
+    directives: [SvgKeyboardWrapComponent, LayersComponent],
```

Comment:
New directives should be added to the NgModule, because adding this way (in the component) is deprecated since rc.5 and we want to avoid the increasing of deprecated code.